components/tabs/demo/tabs.html

```diff
@@ -34,6 +34,40 @@ <h2>Tabs</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Tabs (paired d2l-tab with d2l-panel)</h2>
+
+			<d2l-demo-snippet>
+				<div>This format is not yet functional. Please do not use yet.</div>
+				<template>
+					<d2l-tabs>
+						<d2l-tab id="all" text="All" slot="tabs"></d2l-tab>
```

Comment:
I haven't imported `d2l-tab` yet since I didn't want to try to render anything, so this does nothing at this point.